lightspeed/modules/proc_create-oauth-app.adoc

```diff
@@ -0,0 +1,39 @@
+:_content-type: PROCEDURE
+
+[id="create-oauth-app_{context}"]
+
+= Creating an OAuth application
+
+Use this procedure to create an OAuth application for your {LightspeedShortName} on-premise deployment. 
+
+.Prerequisites
+* You have an operational Ansible automation controller instance.
```

Comment:
use the attribute: You have an operational Ansible {ControllerName} instance.

Reply:
Done